File: app/src/main/java/com/pluu/webtoon/utils/MessageUtils.java
package com.pluu.webtoon.utils;

import android.content.Context;

import com.pluu.webtoon.R;
import com.pluu.webtoon.item.ERROR_TYPE;

/**
 * 메세지 유틸
 * Created by PLUUSYSTEM-NEW on 2016-05-22.
 */
public class MessageUtils {

    public static String getString(Context context, ERROR_TYPE errorType) {
        int resId;
        switch (errorType) {
            case ADULT_CERTIFY:
            case COINT_NEED:
                resId = R.string.msg_not_support;
                break;
            case NOT_SUPPORT:
                resId = R.string.not_support_type;
                break;
            default:
                resId = R.string.network_fail;
        }
        return context.getString(resId);
    }

}

File: app/src/main/java/com/pluu/webtoon/utils/MessageUtils.kt
package com.pluu.webtoon.utils

import android.content.Context

import com.pluu.webtoon.R
import com.pluu.webtoon.item.ERROR_TYPE

/**
 * 메세지 유틸
 * Created by pluu on 2017-04-18.
 */
fun ERROR_TYPE.getMessage(context: Context) : String {
    val resId: Int
    when (this) {
        ERROR_TYPE.ADULT_CERTIFY, ERROR_TYPE.COINT_NEED -> resId = R.string.msg_not_support
        ERROR_TYPE.NOT_SUPPORT -> resId = R.string.not_support_type
        else -> resId = R.string.network_fail
    }
    return context.getString(resId)
}